examples/dashboard.yml

```diff
@@ -36,11 +37,11 @@ sources:
           label: Drop-off Hour
         pickup_hour:
           label: Pick-up Hour
-      plot:
```

Comment:
I'm confused why was this removed?

Reply:
I was using the github UI. Just a mistake